### PR TITLE
feat(issue-platform): Add more profile issue types

### PR DESCRIPTION
**Pull request by phacops**

### CODE REVIEW

File: src/sentry/issues/grouptype.py [hunk 1]
```diff
@@ -144,5 +144,25 @@ class PerformanceUncompressedAssetsGroupType(GroupType):
...
     slug = "profile_blocked_thread"
-    description = "Blocked Thread"
+    category = GroupCategory.PROFILE.value
+
+
+@dataclass(frozen=True)
+class ProfileFileIOGroupType(GroupType):
+    type_id = 2001
+    slug = "profile_file_io_main_thread"
+    category = GroupCategory.PROFILE.value
```
Comment (ceorourke) [inline src/sentry/issues/grouptype.py:154]: Can you add a description field?

Comment (phacops) [reply src/sentry/issues/grouptype.py:154]: Is description really useful when I'm passing the issue title in the occurrence? Do you except something different in the description?

Comment (phacops) [reply src/sentry/issues/grouptype.py:156]: Answered by DM: it will be used in emails or Slack and could be different than issue title.

**Review by ceorourke — APPROVED** [review]

**Review by wedamija — COMMENTED** [review]
Just want to note that at the moment, our process for adding flags to roll out new types is a little painful. Let me know when we get to that point and we can look into adding them.

**Pull request merged**